### ISSUE TITLE
no tests

### ISSUE THREAD

**Issue by ZooeyMiller** [facg3/NewMusicAPI-#17]
There are no tests. At the moment, our code is not very testable, because you have your logic mixed in with your api requests, and your DOM manipulation.

I'd recommend making a separate file for DOM manipulation + API request, and a separate file for your logic, then you can test your logic.

**Comment by MMortaga** [comment]
True. Unfortunately, we've only realized this when it was too late :(
